os/arch/arm/src/armv7-a/arm_syscall.c

```diff
@@ -311,7 +311,7 @@ uint32_t *arm_syscall(uint32_t *regs)
 		DEBUGASSERT(index >= 0);
 		DEBUGASSERT(index < CONFIG_SYS_NNEST);
 		if (index < 0 || index >= CONFIG_SYS_NNEST) {
-			return INDEX_ERROR;
+			svcerr("ERROR: Bad SYS call: 0x%x\n", regs[REG_R0]);
```

Comment:
1. No need to return here? Is it ok to execute next code with invalid index??
2. svcerr uses lldbg. Is it ok? No log mix??